MAVProxy/modules/mavproxy_param.py

```diff
@@ -426,6 +426,7 @@ def cmd_param(self, args):
         '''control parameters'''
         self.check_new_target_system()
         sysid = self.get_sysid()
+        print args
```

Comment:
Is this a debugging statement? Not Python3 compliant. Needs to be ``print(args)``

Reply:
Yes this is a debugging statement. Shouldn’t have been pushed. Will remove it.